tests/test_general.py

```diff
@@ -154,7 +154,8 @@ def _calls_wait_for_calc_timeout_slow(res_queue):
 @pytest.mark.parametrize(
     'mongetter,backend',
     [
-        (_test_mongetter, 'mongo'),
+        # (_test_mongetter, 'mongo'),  # Please do NOT test the mongo backend
```

Comment:
why not rather mark it with
```
pytest.param(_test_mongetter, 'mongo', marks=pytest.mark.mongo)
```
instead of duplicating whole test

Reply:
Oops. I didn't know you're going to review in time. :P
Anyway, I didn't know you could do that with pytest!
Let's try that later. You can work it into your suite separation PR, or later; I'm not sure I understand the exact syntax you refer to.

Reply:
seems that this parameter marking was already used in the tests, see: #145